packages/fx-core/src/component/driver/interface/baseDeployStepDriver.ts

```diff
@@ -26,13 +26,5 @@ export abstract class BaseDeployStepDriver {
     };
   }
 
-  /**
-   * call when error happens
-   * do some resource clean up
-   */
-  async cleanup(): Promise<void> {
```

Comment:
Why do we remove this interface?

Reply:
Because this method has been moved into the `wrapErrorHandler`